source/FreeplayState.hx

```diff
@@ -202,13 +202,27 @@ class FreeplayState extends MusicBeatState
 
 		if (accepted)
 		{
-			trace(StringTools.replace(songs[curSelected].songName," ", "-").toLowerCase());
+			// pre lowercasing the song name (update)
+			var songLowercase = StringTools.replace(songs[curSelected].songName, " ", "-").toLowerCase();
+			switch (songLowercase) {
+				case 'dad-battle': songLowercase = 'dadbattle';
+				case 'philly-nice': songLowercase = 'philly';
+			}
+			// adjusting the highscore song name to be compatible (update)
+			// would read original scores if we didn't change packages
+			var songHighscore = StringTools.replace(songs[curSelected].songName, " ", "-");
+			switch (songHighscore) {
+				case 'Dad-Battle': songHighscore = 'Dadbattle';
+				case 'Philly-Nice': songHighscore = 'Philly';
+			}
```

Comment:
there *has to* be a less janky way to do this but this whole codebase is wack so whatever